src/ducks/SANCharts/Chart/paywall.js

```diff
@@ -0,0 +1,45 @@
+import { getTimeIntervalFromToday, DAY, MONTH } from '../../../utils/dates'
+import { createOptions } from '../../../pages/Chart/paywallBoundaries'
+
+export function drawPaywall (chart, leftBoundaryDate, rightBoundaryDate) {
+  const { ctx, points, left, top, width, height } = chart
+
+  const lastIndex = points.length - 1
+  const firstDate = points[0].value
+  const lastDate = points[lastIndex].value
+
+  ctx.save()
+  ctx.lineWidth = 1
+  ctx.strokeStyle = '#FFAD4D'
```

Comment:
@DmitriVanGuard Do we have this color in our design system?

Reply:
https://storybook.stage.internal.santiment.net/?selectedKind=Colors&selectedStory=List%20of%20all%20colors&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

Reply:
@DmitriVanGuard Before you wrote `ctx.fillStyle` = '#9faac4'` and here you are using capitalize letters

Reply:
`ctx.fillStyle` is not case sensitive